spec/duration.html

```diff
@@ -1564,38 +1564,42 @@ <h1>Duration Nudge Result Records</h1>
       </emu-table>
     </emu-clause>
 
-    <emu-clause id="sec-temporal-nudgetocalendarunit" type="abstract operation">
+    <emu-clause id="sec-temporal-computenudgewindow" type="abstract operation">
       <h1>
-        NudgeToCalendarUnit (
+        ComputeNudgeWindow (
           _sign_: -1 or 1,
           _duration_: an Internal Duration Record,
           _originEpochNs_: a BigInt,
-          _destEpochNs_: a BigInt,
           _isoDateTime_: an ISO Date-Time Record,
           _timeZone_: an available time zone identifier or ~unset~,
           _calendar_: a calendar type,
           _increment_: a positive integer,
           _unit_: a date unit,
-          _roundingMode_: a rounding mode,
-        ): either a normal completion containing a Record with fields [[NudgeResult]] (a Duration Nudge Result Record) and [[Total]] (a mathematical value), or a throw completion
+          _additionalShift_: either *true* or *false*,
+        ): either a normal completion containing a Record with fields [[r1]] (a mathematical value), [[r2]] (a mathematical value), [[startEpochNs_ (a BigInt), _endEpochNs_ (a BigInt), _startDuration_ (an Internal Duration Record), and _endDuration_ (an InternalDurationRecord), or a throw completion
```

Comment:
Record fields are usually initial caps:

```suggestion
        ): either a normal completion containing a Record with fields [[R1]] (a mathematical value), [[R2]] (a mathematical value), [[StartEpochNs]] (a BigInt), [[EndEpochNs]] (a BigInt), [[StartDuration]] (an Internal Duration Record), and [[EndDuration]] (an InternalDurationRecord), or a throw completion
```

(Need to change the field names in the return statement and the caller as well)